.vscode/settings.json

```diff
@@ -27,5 +27,6 @@
   "[html][css][json][yaml][sql][markdown]": {
     "editor.defaultFormatter": "esbenp.prettier-vscode",
     "editor.formatOnSave": true
-  }
+  },
+  "editor.renderWhitespace": "all"
```

Comment:
Please avoid committing changes to this file. I'd generally recommend reviewing your list of changed files locally before committing.

Reply:
done in 05d9c50439e1b31c69e880903d1ece34cfb3e9d4 and I will be more careful about what I add to my commits in the future. thank you for the review!